Source/JavaScriptCore/wasm/WasmFunctionParser.h

```diff
@@ -655,6 +656,10 @@ auto FunctionParser<Context>::simd(SIMDLaneOperation op, SIMDLane lane, SIMDSign
         WASM_PARSER_FAIL_IF(!parseVarUInt32(offset), "can't get simd memory op offset");
 
         WASM_VALIDATOR_FAIL_IF(alignment > maxAlignment, "alignment: ", alignment, " can't be larger than max alignment for simd operation: ", maxAlignment);
+
+        if (!isReachable)
```

Comment:
Use `if constexpr (!isReachable)` instead since `isReachable` is a constant.  Ditto for similar below as well.